### ISSUE TITLE
usdview does not open after build on Windows 10

### ISSUE THREAD

**Issue by abelvargas** [PixarAnimationStudios/USD#1133]
### Description of Issue
After a clean build with no errors, usdview /path/to/usd just waits for a second and does not open at all. All dependecies are installed and environment variables set.
### Steps to Reproduce
1. Build on Windows 10 with no errors
2. usdview /path/to/usd or double click on usdview
3. usdview doesn't open or prints anything at all on the shell. No errors. On double click it opens a blank shell and just closes with no messages.

### System Information (OS, Hardware)
Windows 10, Visual Studio 2019, build_usd.py changed only for virtual memory size (it was failing at the standard value of 150, set to 1000 and it works).

I wish I could give any more information on this error but it won't show anything.
I also followed this: https://tinyhawkus.com/2019/01/04/pixar-usd-on-windows-10/
and it's the same.

**Comment by sunyab** [comment]
Hi @abelvargas, I'm wondering if you are running into https://github.com/PixarAnimationStudios/USD/issues/1095.

**Comment by abelvargas** [comment]
Hey @sunyab, I'm a bit out of my depth here. I'd like to reproduce the error in the thread but I'm not much of a programmer. If it's not a Python code I don't know where to begin. Would you mind explaining to me what do they mean by running that code in a "client app"?

**Comment by ChokmahBinah** [comment]
Hello @abelvargas,

I had the same error.
After adding c:\Python27 to PATH environment variable it worked like a charm.

I hope it could help.

Florian.

**Comment by abelvargas** [comment]
Hey Florian,

Thanks for the reply! c:\Python27 is on both my system and user variables. Along with the ones from USD. Before it was "Path" but I changed the case to "PATH" hoping it would fix it but I still have the same behavior (even after re-starting the machine). Any ideas on how to get a log of what happens when I pass the command and it just doesn't open? I was hoping to read something like "PyQT not found" or something. That would help a lot :)

**Comment by ChokmahBinah** [comment]
Hello @abelvargas ,

Strangely I had the same  problem at the beggining.
Path or PATH means the same thing for windows, it seems not case sensitive.
I setted this vairables too :

![environmentVariables](https://user-images.githubusercontent.com/33164632/76160176-1d840980-6128-11ea-9a46-c40b7d12cb4c.jpg)
 Perhaps it will help you.

As you I know mainly python language (doing tools for Maya) and I am new to building and C++ stuff.
But finally I managed to launch usdview ;)

I'm sure you will manage too :)

**Comment by sunyab** [comment]
Hi @abelvargas, "running that code in a client app" in that case means to compile a C++ program with the code that was mentioned. You can also run that code through the Python interpreter via:
```
from pxr import Plug
Plug.Registry().GetAllPlugins()
```
The top of the bug report describes a workaround for VS2019; if you'd like to try it you would need to modify USD/cmake/defaults/msvcdefaults.cmake in your source tree to change "/Zc:inline" to "/Zc:inline-", then rebuild USD from scratch.

Hope that helps!

**Comment by jilliene** [comment]
Filed as internal issue #USD-5920

**Comment by abelvargas** [comment]
Hello!

Sorry it took me this long to get back to this thread, I had to wait for the weekend! 

@ChokmahBinah I added all the paths and thing's I was missing but it still didn't work, thanks for the help and the print screens, at least that got me to the point where I knew someone had the same paths and it was working :) it's helps to move forward!

@sunyab Thanks for catching the similarities between my issue and the other thread and thank you for explaining the solution and how to fix it. It worked! Rebuilding after changing "/Zc:inline" to "/Zc:inline-" did fix the problem. It's amazing what one little dash will do 👍 

I did keep that little change on the build_usd.py which changed virtual memory size from 150 to 1000 to make sure I wouldn't run into the same issue.

Now on to the next problems! Thanks again
